app/Resources/views/blog/rss.xml.twig

```diff
@@ -0,0 +1,23 @@
+{% spaceless %}
+<?xml version="1.0" encoding="UTF-8" ?>
+<rss version="2.0">
+    <channel>
+        <title>{{ 'rss.title'|trans }}</title>
+        <description>{{ 'rss.description'|trans }}</description>
+        <link>{{ url('blog_index') }}</link>
+        <language>{{ app.request.locale }}</language>
+
+        {% set RSS = constant('RSS', date()) %}
+        {% for post in posts %}
+            <item>
+                <title>{{ post.title }}</title>
+                <description>{{ post.summary }}</description>
+                <link>{{ url('blog_post', {'slug': post.slug}) }}</link>
+                <guid>{{ url('blog_post', {'slug': post.slug}) }}</guid>
+                <pubDate>{{ post.publishedAt|date(format=RSS, timezone='GMT') }}</pubDate>
```

Comment:
Here I also have doubts about:

```twig
{% set RSS = constant('RSS', date()) %}
<pubDate>{{ post.publishedAt|date(format=RSS, timezone='GMT') }}</pubDate>
```

Why not using this?

```twig
<pubDate>{{ post.publishedAt|date('r', timezone='GMT') }}</pubDate>
```

Showing the Twig constants and all it's great ... but we should be careful. If newcomers see this, they'll think that Symfony is this complicated and you must do it like this.